.github/workflows/webpage-preview.yml

```diff
@@ -23,7 +23,68 @@ env:
   GOPROXY: https://proxy.golang.org
 
 jobs:
-  preview:
+  blog-preview:
+    name: Preview blog
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+
+    if: >
+      contains(github.event.pull_request.labels.*.name, 'trust') &&
+      !contains(github.event.pull_request.labels.*.name, 'no ci') &&
+      (contains(github.event.pull_request.labels.*.name, 'blog/engineering') ||
+      contains(github.event.pull_request.labels.*.name, 'blog/marketing'))
```

Comment:
As we have only 2 labels for blogs I've decided to not overcomplicate the check. In the future we might iterate through labels to check if they contain `blog/`